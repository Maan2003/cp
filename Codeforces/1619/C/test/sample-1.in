7
17236 1106911
3465 1106911
1 5
108 112
12345 1023412
1 11
1 20
