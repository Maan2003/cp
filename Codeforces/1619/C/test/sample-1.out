3465
17236
4
-1
90007
10
-1
